css-layout-classes/styles.css

```diff
@@ -0,0 +1,41 @@
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-one-third,
+.column-one-half,
+.column-one-fourth {
+  width: 70%;
+}
+
+.column-one-half-dk-text {
+  width: 100%;
+}
+
+.column-one-half-dk-pic {
+  width: 46%;
+}
+
+.column-one-half-mario-pic {
+  width: 35%;
+}
```

Comment:
Hmmm, the class names here don't need to mention Mario or donkey kong, these are utility classes. Also, the fact that all your column measurements are set to the same size (`70%`), is definitely not correct. Please revise them to have styling that matches their name.